common/src/main/java/org/opentestsystem/rdw/admin/common/security/PermissionScope.java

```diff
@@ -2,13 +2,14 @@
 
 import com.google.common.collect.ImmutableSet;
 
+import java.io.Serializable;
 import java.util.Collection;
 import java.util.Set;
 
 /**
  * Used to limit database query results
  */
-public class PermissionScope {
+public class PermissionScope implements Serializable {
```

Comment:
Ugh, merging this will be annoying since these have been refactored to be shared.